qiskit_experiments/randomized_benchmarking/interleaved_rb_analysis.py

```diff
@@ -108,99 +134,4 @@ def fit_fun_interleaved(x, a, alpha, alpha_c, b):
-            ax = self._generate_multi_errorbar_plot(series, xdata, ydata, ydata_sigma, ax=ax)
-            self._format_plot(ax, analysis_result)
-            ax.legend(loc="center right")
-            figures = [ax.get_figure()]
-        else:
-            figures = None
-        return analysis_result, figures
-
-    @staticmethod
-    def _generate_multi_scatter_plot(series, xdata, ydata, ax):
-        """Generate scatter plot of raw data"""
-        idx0 = series == 0
-        idx1 = series == 1
-        ax = plotting.plot_scatter(xdata[idx0], ydata[idx0], ax=ax)
-        ax = plotting.plot_scatter(xdata[idx1], ydata[idx1], ax=ax, marker="+", c="darkslategrey")
-        return ax
-
-    @staticmethod
-    def _generate_multi_errorbar_plot(series, xdata, ydata, sigma, ax):
-        """Generate errorbar plot of average data"""
-        idx0 = series == 0
-        idx1 = series == 1
-        ax = plotting.plot_errorbar(
-            xdata[idx0],
-            ydata[idx0],
-            sigma[idx0],
-            ax=ax,
-            label="Standard",
-            marker=".",
-            color="red",
-        )
-        ax = plotting.plot_errorbar(
-            xdata[idx1],
-            ydata[idx1],
-            sigma[idx1],
-            ax=ax,
-            label="Interleaved",
-            marker="^",
-            color="orange",
-        )
-        return ax
-
-    @staticmethod
-    def _p0_multi(series, xdata, ydata, num_qubits):
-        """Initial guess for the fitting function"""
-        std_idx = series == 0
-        p0_std = RBAnalysis._p0(xdata[std_idx], ydata[std_idx], num_qubits)
-        int_idx = series == 1
-        p0_int = RBAnalysis._p0(xdata[int_idx], xdata[int_idx], num_qubits)
-        return {
-            "a": np.mean([p0_std["a"], p0_int["a"]]),
-            "alpha": p0_std["alpha"],
-            "alpha_c": min(p0_int["alpha"] / p0_std["alpha"], 1),
-            "b": np.mean([p0_std["b"], p0_int["b"]]),
-        }
-
-    @classmethod
-    def _format_plot(cls, ax, analysis_result, add_label=True):
-        """Format curve fit plot"""
-        # Formatting
-        ax.tick_params(labelsize=14)
-        ax.set_xlabel("Clifford Length", fontsize=16)
-        ax.set_ylabel("Ground State Population", fontsize=16)
-        ax.grid(True)
-
-        if add_label:
-            alpha = analysis_result["popt"][1]
-            alpha_c = analysis_result["popt"][2]
-            alpha_err = analysis_result["popt_err"][1]
-            alpha_c_err = analysis_result["popt_err"][2]
-            epc = analysis_result["EPC"]
-            epc_err = analysis_result["EPC_err"]
-            box_text = "\u03B1:{:.4f} \u00B1 {:.4f}".format(alpha, alpha_err)
-            box_text += "\n\u03B1_c:{:.4f} \u00B1 {:.4f}".format(alpha_c, alpha_c_err)
-            box_text += "\nEPC: {:.4f} \u00B1 {:.4f}".format(epc, epc_err)
-            bbox_props = dict(boxstyle="square,pad=0.3", fc="white", ec="black", lw=1)
```

Comment:
On an aesthetic point I preferred having the fit params in a box within the figure rather than above it.

Reply:
Okey, I reverted the change. However, this hard-coded location can hide curves, especially in sinusoidal fit. Maybe we can implement stylesheet framework (basically for object locations and axis limit control, other properties can be modified with rcparams).